docs/security/trust/trust_sandbox.md

```diff
@@ -327,5 +308,25 @@ Now, that you have a full Docker content trust sandbox on your local system,
 feel free to play with it and see how it behaves. If you find any security
 issues with Docker, feel free to send us an email at <security@docker.com>.
 
+## Troubleshooting
+
+If you see an error like:
+
+    Error: error contacting notary server: dial tcp: lookup notaryserver on 192.168.65.1:53: no such host
+
+or
+
+    Get https://sandboxregistry:5000/v1/_ping: dial tcp: lookup sandboxregistry on 192.168.65.1:53: no such host
+
+Please make sure that your `trustsandbox` container is connected to the notary server and the registry:
 
-&nbsp;
+    $ docker inspect trustsandbox | grep -A 3 Links
+                "Links": [
+                    "/notary_server_1:/trustsandbox/notaryserver"
+                    "/sandboxregistry:/trustsandbox/sandboxregistry"
+                ],
```

Comment:
See above notes w.r.t. links; I don't think you need the links (compose should set up the aliases if the V2 format is used)